### ISSUE TITLE
[filterscripts] coordinate service -> astro (Trac #1641)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1633]
This import needs to change:

```text
  File "/home/dschultz/Documents/combo/trunk/build_memory/lib/icecube/filterscripts/gcfilter.py", line 17, in GCFilter
    from icecube import dataclasses, coordinate_service
ImportError: cannot import name coordinate_service
```

And this line in the file:

```text
add_gcfilter(tray,label=name,If=If,CorsikaMJDStart=coordinate_service.calendar_date_2_mjd(2013,07,01,00,00,00.),CorsikaMJDEnd=coordinate_service.calendar_date_2_mjd(2013,07,01,23,56,04.091)) ##One full MJD
```

Migrated from https://code.icecube.wisc.edu/ticket/1641
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "This import needs to change:\n{{{\n  File \"/home/dschultz/Documents/combo/trunk/build_memory/lib/icecube/filterscripts/gcfilter.py\", line 17, in GCFilter\n    from icecube import dataclasses, coordinate_service\nImportError: cannot import name coordinate_service\n}}}\n\nAnd this line in the file:\n{{{\nadd_gcfilter(tray,label=name,If=If,CorsikaMJDStart=coordinate_service.calendar_date_2_mjd(2013,07,01,00,00,00.),CorsikaMJDEnd=coordinate_service.calendar_date_2_mjd(2013,07,01,23,56,04.091)) ##One full MJD\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[filterscripts] coordinate service -> astro", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-04-17T17:21:55", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160418T08:09:56: 
* kjmeagher commented:

> In changeset 144848:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="144848"
> use `I3Time` instead of coordinate-service to get mjd. fixes #1633
> ```


* kjmeagher changed status from "new" to "closed"
* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
